model/description-model.js

```diff
@@ -0,0 +1,14 @@
+const fs =  require("node:fs/promises");
+
+const fetchDescription = () => {
+    return fs.readFile('./endpoints.json', 'utf-8', (err, data) => {
```

Comment:
If you're using fs/promises, it should be used the right way with a .then() chain.